bed/info_test.go

```diff
@@ -41,3 +41,23 @@ func TestIsSelfOverlapping(t *testing.T) {
 		}
 	}
 }
+
+func TestOverlapSize(t *testing.T) {
```

Comment:
Once you update OverlapLength, you can prob get rid of this

Reply:
Actually, should try to sync with @RavenYL, because there's something similar in PR: #928